receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -128,7 +129,13 @@ func parseMessageToMetric(line string) (*statsDMetric, error) {
 			return nil, fmt.Errorf("unrecognized message part: %s", part)
 		}
 	}
-
+	//fmt.Println("name: " + result.name)
```

Comment:
Can you remove these debugging statements?

Reply:
Sure, will remove